_layouts/home.html

```diff
@@ -10,50 +10,51 @@
   </div>
   {% endif %}
 
-  <div class="posts">
-    <header>
-    <div class="welcome">
-      <h1>Recent Posts</h1>
-    </div>
-  </header>
-    {% for post in paginator.posts %}
-    <div class="post-teaser">
-      <header>
-        <h2>
-          <a class="post-link" href="{{ post.url | relative_url }}">
-            {{ post.title }}
+  <div class="wrapper">
+    <div class="container grid">
+      <main id="main">
+        <h1>Recent Posts</h1>
+        {% for post in paginator.posts %}
+        <div class="post-teaser mb-20">
+          <h2>
+            <a class="post-link" href="{{ post.url | relative_url }}">
+              {{ post.title }}
+            </a>
+          </h2>
+          <p class="meta">
+            {{ post.date | date: "%B %-d, %Y" }}
+          </p>
+          <div class="excerpt pb-10">
+            {{ post.excerpt }}
+          </div>
+          <div class="continue-reading">
+            <a href="{{ post.url | relative_url }}" aria-label="{{ post.title | escape }}" class="btn btn--sm btn--transparent">
+              {{ site.theme_settings.str_continue_reading }}
+            </a>
+          </div>
+        </div>
+        <hr/>
+        {% endfor %}
+
+        {% if paginator.total_pages > 1 %}
+        <div class="pagination">
+          {% if paginator.previous_page %}
+          <a href="{{ paginator.previous_page_path | relative_url | replace: '//', '/' }}" class="btn btn--lg btn--blue">
+            <i class="fa fa-chevron-left"></i>
+            {{ site.theme_settings.str_previous_page }}
+          </a>
+          {% endif %}
+          {% if paginator.next_page %}
+          <a href="{{ paginator.next_page_path | relative_url | replace: '//', '/' }}" class="btn btn--lg btn--blue ml-60">
+            {{ site.theme_settings.str_next_page }}
+            <i class="fa fa-chevron-right"></i>
```

Comment:
Instead of using the `<i>` element here, I suggest following the [Style Guide pattern for adding icons to buttons](https://styles.rockarch.org/?path=/story/components-button--icons) using material icons. It looks like the material icons we want here are "[chevron_right](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:chevron_right:FILL@0;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%235f6368)" and "[chevron_left](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:chevron_left:FILL@0;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%235f6368)".